modules/rte_mis_mail/rte_mis_mail.module

```diff
@@ -41,6 +41,7 @@ function rte_mis_mail_form_user_form_alter(&$form, $form_state) {
       '#attributes' => [
         'id' => ['email-container'],
       ],
+      '#weight' => array_key_exists('field_phone_number', $form) ?? $form['field_phone_number']['#weight'] - 1,
```

Comment:
This will return TRUE when field **field_phone_number** is present  `array_key_exists('field_phone_number', $form)`

Reply:
Yes, wanted to check if the phone number field exists or not. If it exists, we would add the email container above the phone number field.

Reply:
Then please use the ternary operator as Null Coalescing Operator will not refer to second operator if first parameter is present. Ref - https://www.php.net/manual/en/migration70.new-features.php#migration70.new-features.null-coalesce-op

Reply:
ok